charts/yet-another-cloudwatch-exporter/templates/deployment.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "yet-another-cloudwatch-exporter.labels" . | nindent 4 }}
+  annotations:
+    {{- with .Values.podAnnotations }}
```

Comment:
Can you please move this _before_ `annotations` so that the section is not rendered at all if there are no annotations to be added?